### PR TITLE
Add autopay to quotes

### PR DESCRIPTION
**Pull request by swilczynski**
## Summary
This PR adds autopay option to quotes, to be used when creating, or modifying related subscription.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/QuoteChangeOrder.yaml [hunk 1]
```diff
@@ -378,6 +378,12 @@ properties:
                 - 'number'
                 - 'null'
               format: double
+  autopay:
+    type:
+      - 'boolean'
+      - 'null'
+    default: null
```
Comment (ikram-patah) [inline openapi/components/schemas/QuoteChangeOrder.yaml:385]: question: Should it not default to `false`?

File: openapi/components/schemas/QuoteReactivateOrder.yaml [hunk 2]
```diff
@@ -384,6 +384,12 @@ properties:
                 - 'number'
                 - 'null'
               format: double
+  autopay:
+    type:
+      - 'boolean'
+      - 'null'
+    default: null
```
Comment (ikram-patah) [inline openapi/components/schemas/QuoteReactivateOrder.yaml:391]: question: same as above (default to `false`)?

File: openapi/components/schemas/QuoteChangeOrder.yaml (hunk 1 above)
Comment (lexerom) [reply openapi/components/schemas/QuoteChangeOrder.yaml:385]: No, if not provided we don't change existing order autopay behavior.

Comment (ZengJunyong) [inline openapi/components/schemas/QuoteChangeOrder.yaml:385]: - default value is `false`? 
- and type is `boolean`(remove `null` type) is enough?

File: openapi/components/schemas/QuoteCreateOrder.yaml [hunk 3]
```diff
@@ -364,6 +364,12 @@ properties:
                 - 'number'
                 - 'null'
               format: double
+  autopay:
+    type:
+      - 'boolean'
+      - 'null'
+    default: false
```
Comment (ZengJunyong) [inline openapi/components/schemas/QuoteCreateOrder.yaml:371]: why default value is `false` while the other 2 are `null`?

Comment (lexerom) [reply openapi/components/schemas/QuoteCreateOrder.yaml:371]: It should be strict `bool` for Creation Quotes. We always provide `autopay` param when creating a new order. For change and reactivate the related order already has some value there.

File: openapi/components/schemas/QuoteCreateOrder.yaml [hunk 4]
```diff
@@ -364,6 +364,12 @@ properties:
                 - 'number'
                 - 'null'
               format: double
+  autopay:
+    type:
+      - 'boolean'
+      - 'null'
```
Comment (lexerom) [inline openapi/components/schemas/QuoteCreateOrder.yaml:370]:
```suggestion
```

Comment (lexerom) [inline openapi/components/schemas/QuoteCreateOrder.yaml:370]:
```suggestion
    type: boolean
```

**Review by lexerom — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by ZengJunyong — APPROVED** [review]

**Review by olasunkanmi04 — APPROVED** [review]

**Review by i-yasko — APPROVED** [review]

**Pull request merged**